src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Reload/ProjectReloadManager.cs

```diff
@@ -25,7 +25,7 @@ namespace Microsoft.VisualStudio.ProjectSystem.VS
     /// projects which fail the solution level reload, the next time the project file changes.
     /// </summary>
     [Export(typeof(IProjectReloadManager))]
-    [AppliesTo("HandlesOwnReload")] // Replace with ProjectCapabilities.HandlesOwnReload when new CPS is available
+    [AppliesTo(ProjectCapabilities.HandlesOwnReload)]
```

Comment:
We've been copying everything over to https://github.com/dotnet/roslyn-project-system/blob/master/src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/ProjectCapability.cs. Why stop here?